epub33/core/index.html

```diff
@@ -4445,10 +4445,17 @@ <h4>Scripting Contexts</h4>
 						</dd>
 					</dl>
 
-					<p>In both of the above-defined contexts, whether the JavaScript code is embedded directly in the
+					<p>In both of the above-defined contexts, whether the code is embedded directly in the
 							<code>script</code> element or referenced via its <code>src</code> attribute makes no
 						difference to the executing context.</p>
 
+					<div class="note">
+						<p>When an [[HTML]] <code>script</code> element contains a <a
+								href="https://html.spec.whatwg.org/multipage/scripting.html#data-block">data block</a>
+							[[HTML]], it does not represent scripted content. [[SVG]] does not define data blocks as of
+							publication, but the same exclusion would apply to a future update.</p>
+					</div>
+
```

Comment:
To be extremely picky (sorry...) the first sentence should not be in a Note, which is informational. That should be part of the normative text imho...

Reply:
Ya, it felt kind of icky when I was adding it. No normative keywords, but it's fully a note, either.